### PR TITLE
Feature/orca 561 Add GraphQL image to new AWS task and service definition

### PR DESCRIPTION
**Pull request by AndrewDorn**
## Summary of Changes

GraphQL image is now deployed and run as an ECS service.

Addresses [ORCA-561: Add GraphQL image to new AWS task and service definition](https://bugs.earthdata.nasa.gov/browse/ORCA-561)

## Changes

* Added service to TF

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Deployed to AWS. Functionality largely untested; task for upcoming card.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [x] CHANGELOG.md
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
    - [x] Automated tests passing (if not fork)
- [x] Any dependent changes have been merged and published in downstream modules
- [x] My code has passed security scanning
    - [x] Snyk
    - [x] git-secrets

### CODE REVIEW

File: modules/graph_ql/main.tf [hunk 1]
```diff
@@ -68,4 +77,17 @@ resource "aws_ecs_task_definition" "graphql_task" {
   }
 ]
 DEFINITION
+}
+
+resource "aws_ecs_service" "graphql_service" {
+  name            = "${var.prefix}_graphql_service"
+  cluster         = var.ecs_cluster_id
+  task_definition = aws_ecs_task_definition.graphql_task.arn
+  desired_count   = 3
```
Comment (rizbihassan) [inline modules/graph_ql/main.tf:86]: this means 3 instances will keep running all time. Should we lower this to 2 to save cost?

**Comment by AndrewDorn** [comment]
> Looks reasonable.
> 
> I think you also need to update the main.tf and variables.tf to include the new required variables.

I have already updated these files, and tested by deploying to AWS. Which main.tf and variables.tf are you referring to?

**Comment by rizbihassan** [comment]
> > Looks reasonable.
> > I think you also need to update the main.tf and variables.tf to include the new required variables.
> 
> I have already updated these files, and tested by deploying to AWS. Which main.tf and variables.tf are you referring to?

I was referring to cumulus-orca/main.tf and cumulus-orca/variables.tf

**Comment by AndrewDorn** [comment]
> > > Looks reasonable.
> > > I think you also need to update the main.tf and variables.tf to include the new required variables.
> > 
> > 
> > I have already updated these files, and tested by deploying to AWS. Which main.tf and variables.tf are you referring to?
> 
> I was referring to cumulus-orca/main.tf and cumulus-orca/variables.tf

Gotcha. I have no new variables at that level.

File: modules/graph_ql/main.tf (hunk 1 above)
Comment (bhazuka) [reply modules/graph_ql/main.tf:86]: 3 is good, you always want an odd number to maintain availability.

**Review by bhazuka — APPROVED** [review]

**Review by rizbihassan — APPROVED** [review]

**Pull request merged**